### ISSUE TITLE
Firmware missing for Intel Dual Band Wireless-9560

### ISSUE THREAD

**Issue by thundron** [balena-os/balena-intel#182]
The Intel Dual Band Wireless-AC 9560 firmware was supposed to be included in v2.27, but it looks like we did not do that, as we did not properly manage the flow of the previously reported issue.
The needed firmware is `iwlwifi-9000-pu-b0-jf-b0-34.ucode`

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_gdc3)

**Comment by pdcastro** [comment]
I understand that work on this ticket is blocked by another ticket:
https://github.com/balena-os/meta-balena/issues/1366

That other ticket has the purpose of freeing up space in the relevant system partition so that more firmwares can be shipped for the NUC with the currently released OS versions (I understand that balenaOS 2.29 is also affected). balenaOS 3.0 will attack the problem from a different angle: increasing the partition size.

**Comment by pdcastro** [comment]
Forum thread reporting the problem on 2.29.0+rev1:
https://forums.balena.io/t/intel-nuc-wifi-doesnt-work/5041/6

**Comment by floion** [comment]
Fixed by https://github.com/balena-os/balena-intel/pull/187

**Comment by floion** [comment]
Closing issue as the firmware will be available in release v2.29.2+rev2 or newer
